docs/userGuide/contentAuthoring.md

```diff
@@ -300,19 +306,20 @@ If you need additional footers for different pages, you can create footer files
 </footer>
 ```
 
-After which, you must specify the file within [front matter](#front-matter)'s `footer` attribute to render the footer.
+After which, you must specify the file within a page's [front matter](#front-matter)'s `footer` attribute to render the footer.

```

Comment:
Maybe just `page front matter's` (no need `'s` after `page`, looks weird to have two `'s` appearing).